api/client/client.go

```diff
@@ -212,12 +212,13 @@ func (c *Client) UploadDirectory(dir, defaultPath, manifest string, toEncrypt, t
 			return "", fmt.Errorf("default path: %v", err)
 		}
 	}
-	return c.TarUpload(manifest, &DirectoryUploader{dir}, defaultPath, toEncrypt, toPin, anonymous, publisher)
+	return c.TarUpload(manifest, &DirectoryUploader{dir}, defaultPath, toEncrypt, toPin, anonymous)
 }
 
 // DownloadDirectory downloads the files contained in a swarm manifest under
 // the given path into a local directory (existing files will be overwritten)
-func (c *Client) DownloadDirectory(hash, path, destDir, credentials string) error {
+func (c *Client) DownloadDirectory(hash, path, destDir, credentials, publisher string) error {
+	log.Debug("DownloadDirectory", "publisher", publisher)
```

Comment:
are you sure you want to leave this `Debug`?

if so, please add text to it to make it more descriptive